notebooks/official/migration/UJ1 Vertex SDK AutoML Image Classification.ipynb

```diff
@@ -0,0 +1,1425 @@
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "deploy_model:mbsdk,automatic"
+      },
+      "source": [
+        "*Example output:*\n",
+        "\n",
+        "    INFO:google.cloud.aiplatform.models:Creating Endpoint\n",
+        "    INFO:google.cloud.aiplatform.models:Create Endpoint backing LRO: projects/759209241365/locations/us-central1/endpoints/4867177336350441472/operations/4087251132693348352\n",
+        "    INFO:google.cloud.aiplatform.models:Endpoint created. Resource name: projects/759209241365/locations/us-central1/endpoints/4867177336350441472\n",
+        "    INFO:google.cloud.aiplatform.models:To use this Endpoint in another session:\n",
+        "    INFO:google.cloud.aiplatform.models:endpoint = aiplatform.Endpoint('projects/759209241365/locations/us-central1/endpoints/4867177336350441472')\n",
+        "    INFO:google.cloud.aiplatform.models:Deploying model to Endpoint : projects/759209241365/locations/us-central1/endpoints/4867177336350441472\n",
+        "    INFO:google.cloud.aiplatform.models:Deploy Endpoint model backing LRO: projects/759209241365/locations/us-central1/endpoints/4867177336350441472/operations/1691336130932244480\n",
+        "    INFO:google.cloud.aiplatform.models:Endpoint model deployed. Resource name: projects/759209241365/locations/us-central1/endpoints/4867177336350441472"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "endpoints_predict:migration,new,mbsdk"
+      },
+      "source": [
+        "### [predictions.online-prediction-automl](https://cloud.google.com/vertex-ai/docs/predictions/online-predictions-automl)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "get_test_item"
+      },
+      "source": [
+        "### Get test item\n",
+        "\n",
+        "You will use an arbitrary example out of the dataset as a test item. Don't be concerned that the example was likely used in training the model -- we just want to demonstrate how to make a prediction."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "get_test_item:automl,icn,csv"
+      },
+      "outputs": [],
+      "source": [
+        "test_item = !gsutil cat $IMPORT_FILE | head -n1\n",
+        "if len(str(test_item[0]).split(\",\")) == 3:\n",
+        "    _, test_item, test_label = str(test_item[0]).split(\",\")\n",
+        "else:\n",
+        "    test_item, test_label = str(test_item[0]).split(\",\")\n",
+        "\n",
+        "print(test_item, test_label)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "predict_request:mbsdk,icn"
+      },
+      "source": [
+        "### Make the prediction\n",
+        "\n",
+        "Now that your `Model` resource is deployed to an `Endpoint` resource, one can do online predictions by sending prediction requests to the Endpoint resource.\n",
```

Comment:
Nit: replace "one" with "you" to make it:

...an `Endpoint` resource, you can do online predictions...